client/src/app/modules/login/components/login.component.html

```diff
@@ -0,0 +1,66 @@
+<div class="page">
+  <ng-container *ngIf="isLoading">
+    <app-loading-indicator></app-loading-indicator>
+  </ng-container>
+
+  <form *ngIf="!isLoading" [formGroup]="form" (ngSubmit)="intendFormSubmit()" novalidate id="form">
+  <div class="box">
+    <div class="page__header">
+      <h3 class="mat-h3">Login</h3>
+    </div>
+    <div class="page__content">
+      <div class="row">
```

Comment:
I'm wondering, if we really need the wrapping divs with class ``row``. Would it also work like this:
```
<div class="page__content col">
  <div class="col">...username-related stuff...</div>
  <div class="col">...password-related stuff...</div>
</div>
```

Reply:
Actually it works also without the `col` at all as they are just 2 form fields below each other.